front/app/component-library/components/Icon/Svg.tsx

```diff
@@ -59,9 +69,11 @@ const Svg = ({
       width={width}
       fill={fill}
       transform={transform}
+      aria-labelledby={titleId}
+      name={name}
```

Comment:
Curious why we're adding `name` here. Could you add a comment?

Reply:
`name` on svg is deprecated and seemingly used for different purposes: https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/name